corehq/apps/reports/dispatcher.py

```diff
@@ -84,7 +87,19 @@ def process(reports):
             return tuple(reports)
 
         corehq_reports = process(getattr(reports, attr_name, ()))
-        custom_reports = process(getattr(domain_module, attr_name, ()))
+
+        module_name = get_domain_module_map().get(domain)
+        if module_name is None:
+            custom_reports = ()
+        else:
+            module = __import__(module_name, fromlist=['reports'])
+            reports = getattr(module, 'reports')
+            custom_reports = process(getattr(reports, attr_name, ()))
+
+            # soon to be removed
```

Comment:
@nickpell I think that this can be removed now right?

Reply:
Looks like yes.